release.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python2.7
+
+import semver
+import xml.etree.ElementTree as ET
+import sys 
+
+"""Type: Major/Minor/Patch"""
+
+"""
+1. Read in values.xml
+2. Increment the SDK version value based on correct semVers
+3. Return: new version number
+"""
+def get_current_version(root):
+  for sdk_version in root.iter("sdk_version"):
+    return sdk_version.text
+
+def update_version(root, xml, version):
+  for sdk_version in root.iter("sdk_version"):
+    sdk_version.text = str(version)
+    sdk_version.set('updated', 'yes')
+    xml.write("AndroidSDK/res/values/strings.xml")
+
+def main():
+  type = sys.argv[1]
+  xml = ET.parse("AndroidSDK/res/values/strings.xml")
```

Comment:
All "magic" strings should be extracted to static variables. Especially if they are used in multiple places.

Reply:
done, open to naming suggestions. i went with `android_strings_xml`